include/zephyr/drivers/clock_control/clock_control_tcc_ccu.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 Hounjoung Rim <hounjoung@tsnlab.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#ifndef CLOCK_CONTROL_TCC_CCU_HEADER
+#define CLOCK_CONTROL_TCC_CCU_HEADER
+
+/*
+ * DEFINITIONS
+ *
+ */
```

Comment:
```suggestion
```

Reply:
I don't know the difference between the two. There appears to be no difference when I copy and paste the part you suggested.